FpsCheck.py

```diff
@@ -342,6 +340,15 @@ def check_list_sites(self, primary, site_list):
                     "Experienced an error when trying to access " + url + "; "
                     + "error was: " + str(inst))
     
+    def check_well_known_list(self, field, list1, list2):
+        """Checks that 2 lists for a given field match each other"""
+        if list1 == list2:
+            return []
+        diff = sorted(set(list1) ^ set(list2))
+        return ["Experienced a mismatch between the PR submission and the " 
+        + WELL_KNOWN + " file:\n\t" + field + " was " + str(list1) + " in the PR, and "
+        + str(list2) + " in the well-known.\n\tDiff was: " + str(diff) + "."]
+
     def find_invalid_well_known(self, check_sets):
         """Checks for and validates well-known pages for FPS sets
```

Comment:
Q: if the checks of a .well-known file fail, do the error messages make it clear to the author _which_ .well-known file they have to fix?

Reply:
Errors messages for the non-primaries mention the site name, but for the primary they do not.

Reply:
Ack. Doesn't need to be part of this PR, but I think it'd be good if we included the info when checking the primary .well-known as well, so that it's very clear where the errors came from.

Reply:
We have to understand where th error is coming from .it's a big frustration